yeat/cli/cli.py

```diff
@@ -72,37 +76,24 @@ def options(parser):
         default=1,
         help="execute workflow with T threads; by default T=1",
     )
-    parser.add_argument("-v", "--version", action="version", version=f"YEAT v{yeat.__version__}")
-
-
-def read_input_types(parser):
-    read_input_options = parser.add_argument_group("input read options")
-    mx = read_input_options.add_mutually_exclusive_group(required=True)
-    mx.add_argument(
-        "--paired",
-        metavar=("READ1", "READ2"),
-        type=str,
-        nargs=2,
-        help="paired-end reads in FASTQ format",
+    parser.add_argument(
+        "-n",
+        "--dry-run",
+        action="store_true",
+        help="construct workflow DAG and print a summary but do not execute",
     )
-    mx.add_argument("--pacbio", metavar="READ", type=str, help="PacBio HiFi-reads in FASTQ format")
 
 
 def get_parser(exit_on_error=True):
     parser = ArgumentParser(exit_on_error=exit_on_error)
     options(parser)
     downsample.options(parser)
-    read_input_types(parser)
     parser.add_argument("config", type=str, help="config file")
     return parser
 
 
 def main(args=None):
     if args is None:
-        args = get_parser().parse_args()
-    assembly_configs = AssemblerConfig.parse_json(open(args.config))
-    workflows.run_workflow(args, assembly_configs)
-    if not args.dry_run:
-        bandage.run_bandage(
-            assembly_configs=assembly_configs, outdir=args.outdir, cores=args.threads
-        )
```

Comment:
Moved the entire "bandage" workflow into the `workflows` directory.

`bandage.run_bandage()` will be called in the `run_workflows()` function.